apis/config/v1beta1/configuration_types.go

```diff
@@ -387,6 +387,25 @@ type Integrations struct {
 	LabelKeysToCopy []string `json:"labelKeysToCopy,omitempty"`
 }
 
+type IntegrationReference string
+
+const (
+	BatchJob        IntegrationReference = "batch/job"
+	RayJob          IntegrationReference = "ray.io/rayjob"
+	RayCluster      IntegrationReference = "ray.io/raycluster"
+	JobSet          IntegrationReference = "jobset.x-k8s.io/jobset"
+	MPIJob          IntegrationReference = "kubeflow.org/mpijob"
+	PaddleJob       IntegrationReference = "kubeflow.org/paddlejob"
+	PyTorchJob      IntegrationReference = "kubeflow.org/pytorchjob"
+	TFJob           IntegrationReference = "kubeflow.org/tfjob"
+	XGBoostJob      IntegrationReference = "kubeflow.org/xgboostjob"
+	AppWrappers     IntegrationReference = "workload.codeflare.dev/appwrapper"
```

Comment:
Suggest making this singular (AppWrapper) instead of plural, to match all the other names (ie its PyTorchJob, not PyTorchJobs).

Reply:
good call. Updated.